src/OpenTK.Windowing.Desktop/GameWindow.cs

```diff
@@ -195,17 +184,43 @@ public double UpdateFrequency
         public GameWindow(GameWindowSettings gameWindowSettings, NativeWindowSettings nativeWindowSettings)
             : base(nativeWindowSettings)
         {
-            IsMultiThreaded = gameWindowSettings.IsMultiThreaded;
-
-            RenderFrequency = gameWindowSettings.RenderFrequency;
             UpdateFrequency = gameWindowSettings.UpdateFrequency;
         }
 
+        #region Win32 Function for timing
+
+        [DllImport("kernel32", SetLastError = true)]
+        private static extern IntPtr SetThreadAffinityMask(IntPtr hThread, IntPtr dwThreadAffinityMask);
+
+        [DllImport("kernel32")]
+        private static extern IntPtr GetCurrentThread();
+
+        [DllImport("winmm")]
+        private static extern uint timeBeginPeriod(uint uPeriod);
+
+        [DllImport("winmm")]
+        private static extern uint timeEndPeriod(uint uPeriod);
+
+        #endregion
+
+        // Counter for how many updates in Run() where slow.
+        private int _slowUpdates = 0;
```

Comment:
How are we defining 'slow'? Can we include a definition here?
Also, let's make this a `///` comment instead - no need for summary blocks or anything.